generate.js

```diff
@@ -45,7 +45,7 @@ const typescriptTemplate = {
   dir: 'app-ts',
   main: 'app.ts',
   scripts: {
-    test: 'npm run build:ts && tsc -p test/tsconfig.test.json && tap test/**/*.test.ts',
+    test: 'tap test/**/*.test.ts',
```

Comment:
Here we want to be sure that the project builds.
```suggestion
    test: 'npm run build:ts && tap test/**/*.test.ts',
```